src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -106,6 +110,14 @@ object DepositHandler {
           _ <- props.setState(INVALID, msg)
           _ <- props.save()
         } yield ()
+      case RejectedDepositException(_, msg, cause) =>
+        log.error(s"[$id] Rejected deposit", cause)
+        for {
+          _ <- removeZipFiles(depositDir) // Currently, the only reason for SWORD 2 to reject a deposit, is insufficient disk space, so we clean up, here.
+          props <- DepositProperties(id)
```

Comment:
I see code duplication with the case above. Only difference is the `INVALID`/`REJECTED` state. Can we make a method out of this?